project_proposals/NGE.md

```diff
@@ -0,0 +1,190 @@
+* lgpl_v2_1   : GNU General Lesser Public License (LGPL) version 2.1
+* lgpl_v3     : GNU General Lesser Public License (LGPL) version 3.0
+* mit         : MIT-License
+```
+
+
+
+## Are any trademarks associated with the project?
+None
+
+
+## Does the project have a web site? Where is it? Are you ok with moving it to be hosted by the OpenRail Association?
+[GITHUB: README](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/blob/main/README.md)
+
+
+## What are the communication channels the project uses (such as mailing lists, Slack, IRC, etc.)?
+Mail, Github (Issue)
+
+
+## What is the project's leadership team and decision-making process?
+tbd
+
+## How is it decided if and when a pull request is merged?
+Pull requests are reviewed and merged by the maintainers.
+
+## How can someone become a committer or a maintainer to/of the project?
+Contributor: via Issues; pull requests - Maintainer tbd
+
+## How is development of the project planned and organized? Is this transparent to the public?
+tbd
+
+
+## What is the project's roadmap?
+[Roadmap](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/blob/main/ROADMAP.md)
+
+
+## What other organizations in the world should be interested in this project?
+The versatility of the editor makes it suitable for various logistics planning scenarios, enabling
+efficient timetable creation and analysis. The software's key features can be leveraged in a broader
+context, such as:
+ 
+- **Public Transportation Systems:** Other public transport networks can utilize the editor's
+  flexibility and features to effectively visualize and plan their transportation systems,
+  leading to improved efficiency and better services.
+ 
+- **Educations:** The editor's capabilities can be utilized in educational institutions to demonstrate
+  how to create integrated timetables and what properties need to be considered during the planning
+  process in order to enable efficient scheduling
+
+
+## Why would this project be a good candidate for inclusion in the OpenRail Association?
+- With its clear focus on the rapid recording of timetable variants on a macroscopic level, Netzgrafik-Editor is suitable as a tool that can be combined with other OSS tools in the area of timetable planning and simulation (e.g., OSRD; MATsim).
+- The Open Rail Association can provide support in building up members who are actively involved in the project community and in defining community governance.
+
+
+## Are there competing products or projects? If there are please explain how the proposed projects differentiates.
+- With this scope on the macroscopic not known.
+
+
+## What standards does the project implement or rely on? How are they related to other existing standards?
+None
+
+
+## What is the tech stack of the project? Name the major programming languages and frameworks which are used.
+[angular.sbb.ch](https://angular.app.sbb.ch/)
```

Comment:
It would be good to mention the tech stack of the backend as well (Spring Boot with PostgreSQL backend)